tests/test_gamma.py

```diff
@@ -1,53 +1,54 @@
-import unittest
-import bayem
-from hypothesis import given, settings
 import hypothesis.strategies as st
+import pytest
+from hypothesis import given, settings
+
+import bayem
+
 
+def test_print():
+    print(bayem.Gamma(42, 2))
 
-class TestGamma(unittest.TestCase):
-    def test_print(self):
-        print(bayem.Gamma(42, 2))
-
-    def test_sd(self):
-        scale, shape = 42, 6174
-        gamma = bayem.Gamma(shape=shape, scale=scale)
-        self.assertAlmostEqual(gamma.mean, shape * scale)
-        variance = shape * scale ** 2
-        self.assertAlmostEqual(gamma.std, variance ** 0.5)
-
-    def test_from_mean_and_sd(self):
-        gamma = bayem.Gamma.FromMeanStd(6174, 42)
-        self.assertAlmostEqual(gamma.mean, 6174)
-        self.assertAlmostEqual(gamma.std, 42)
-
-    @settings(derandomize=True, max_examples=200)
-    @given(
-        st.tuples(
-            st.floats(min_value=1.0e-4, max_value=1e4),
-            st.floats(min_value=1.0e-4, max_value=1e4),
-        )
+
+def test_sd():
+    scale, shape = 42, 6174
+    gamma = bayem.Gamma(shape=shape, scale=scale)
+
+    assert gamma.mean == pytest.approx(shape * scale)
+    variance = shape * scale ** 2
+    assert gamma.std == pytest.approx(variance ** 0.5)
```

Comment:
you could also test gamma.std**2 == , thus avoiding the additional variance line and computing the sqrt